include/cv/localization.hpp

```diff
@@ -81,6 +84,8 @@ class ECEFLocalization : Localization {
 class GSDLocalization : Localization {
  public:
     GPSCoord localize(const ImageTelemetry& telemetry, const Bbox& targetBbox) override;
+    GPSCoord CalcOffset(const double offset_x, const double offset_y, const double lat, const double lon);
```

Comment:
nitpick: I wouldn't hate a more descriptive name instead of `CalcOffset` but it's not that big of a deal.